thing_shots.c

```diff
@@ -0,0 +1,1408 @@
+/******************************************************************************/
```

Comment:
Why is this entire file shown as new, instead of just the changes visible?

Reply:
Must be because of some line changes affecting the whole document, but it would see that's a silly thing for github to show. When I compare the two files using sh.exe with the command diff -b thing_shots.c thing_shots_new.c I get the following actual changes:

125a126
>       case ShM_Firebomb:
799c800,801
<     throw_strength = shotng->field_20;
---
>     //throw_strength = shotng->field_20; //this seems to be always 0, this is why it didn't work;
>       throw_strength = shotst->old->push_on_hit;
877c879,880
<     amp = shotng->field_20;
---
>       //amp = shotng->field_20;
>       amp = shotst->old->push_on_hit;
988c991
<         i = amp * (long)shotng->velocity.x.val;
---
>               amp * (long)shotng->velocity.x.val;
990c993
<         i = amp * (long)shotng->velocity.y.val;
---
>               amp * (long)shotng->velocity.y.val;
1009a1013,1015
>       if (shotst->area_range != 0) detonate_shot(shotng);

Reply:
Must be because of line differences that affected the whole document, but that's silly for github to show the changes in that way. When I run sh.exe and use the command diff -b thing_shots.c thing_shots_new.c I get the following actual differences:

https://i.imgur.com/WldHorD.png

Reply:
For something a little easier to read like the way github outputs it, and with some context for each change:

[thing_shots_diff.txt](https://github.com/Loobinex/keeperfx-unofficial/files/1735128/thing_shots_diff.txt)